sdk/messaging/azservicebus/message.go

```diff
@@ -4,105 +4,331 @@
 package azservicebus
 
 import (
+	"context"
+	"fmt"
+	"time"
+
 	"github.com/Azure/azure-amqp-common-go/v3/uuid"
-	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal"
 	"github.com/Azure/go-amqp"
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
This should be removed.  If there is a large need you can use `azcore/to` instead.

Reply:
Removed all autorest/to usages, replaced with azcore/to for now.